### ISSUE TITLE
Pretraining Configuration

### ISSUE THREAD

**Issue by vateye** [feizc/Visual-LLaMA#1]
Hi, I saw you use CC, COCO and COYO-700M, so the actual training data you used is about 700M image-text pairs? Did you train on those data for total 10 epochs (i.e., 7B samples)? And your batch size is only 64?

**Comment by feizc** [comment]
Total dataset is ～0.7B image-text pairs (actually less than it as some url are missing) for connection network learning. You can find that the loss reduction is very stable ：）

**Comment by vateye** [comment]
So you run it for about 10 epochs? How much time did it cost? and machine configuration?

**Comment by vateye** [comment]
Besides, according to your implementation, the "[boi] and [eoi]" would not be updated if you freeze the whole LLM. How did you solve it?
